app/base/static/property_images/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+*/
```

Comment:
in addition I still dont understand what you mean (or what you  want git to ignore) when you say `*/`
to me that would mean ignore every directory in this project and dont track it by git
is that so

Reply:
I want the `property_images` and the `profile_pictures` directories to be included as empty folders are excluded from the repository.